index.js

```diff
@@ -53,7 +55,7 @@ Twig.extend(function(Twig) {
 
         if (includes.length > 0) {
             _.each(_.uniq(includes), function(file) {
-                output.unshift("require("+ JSON.stringify("twig!" + file) +");\n");
+                output.unshift("require("+ JSON.stringify(file) +");\n");
```

Comment:
I'm not convinced this is a good solution. If we remove the prefix, we will force users to add config which is not very conventional. I must investigate further why the addition of this prefix is creating duplicate modules...